sdk/eventhub/event-hubs/changelog.md

```diff
@@ -1,3 +1,28 @@
+### 2019-08-06 5.0.0-preview.2
+
+#### General
+- The current `Sender` does not have a provision of "awaiting" on sending a message. We need to add handlers on the `Sender` for accepted, rejected, released, modified to ensure whether the message was successfully sent. 
+  Now, we use a new `AwaitableSender` instead of `Sender` which adds the handlers internally and provides an awaitable `send()` operation.
```

Comment:
I would rephrase the whole `awaitable sender` work to just say we made performance improvements to the `send()` method that reduces the instances of TooManyEventListeners warning.

Reply:
> The sender is refactored to avoid the `TooManyEventListeners` warning which would occur before if too many send requests were in flight at the same time from the same sender

Confirm the name of the warning